stock_picking_reorder_lines/views/view.xml

```diff
@@ -3,7 +3,6 @@
     <data>
 
         <record id="view_move_picking_tree" model="ir.ui.view">
-            <field name="name">stock.move.tree</field>
```

Comment:
@legalsylvain This name will be displayed in Settings -> ... -> Views you can see it from the interface